README.md

```diff
@@ -97,7 +118,7 @@ The table below shows the track parameters that can be adjusted along with defau
 
 ### elasticlogs-continuous-index-and-query
 
-This challenge is suitable for long term execution and runs in two phases. Both phases (`p1`, `p2`) index documents containing auto-generated event, however, `p1` indexes events at the max possible speed, whereas `p2` throttles indexing to a specified rate and in parallel executes four queries simulating Kibana dashboards and queries. The created index gets rolled over after the configured max size and the maximum amount of rolled over indices are also configurable.
+This challenge is suitable for long term execution and runs in two phases. Both phases (`p1`, `p2`) index documents containing auto-generated events, however, `p1` indexes events at the max possible speed, whereas `p2` throttles indexing to a specified rate and in parallel executes four queries simulating Kibana dashboards and queries. The created index gets rolled over after the configured max size.  The maximum amount of rolled over indices are also configurable.
```

Comment:
nit: There are two whitespaces between the period and `The maximum amount of rolled over indices ...`

Reply:
Oh do we usually do 1?  I am used to writing 2 white spaces after a period.

Reply:
Right we use 1 for all text in related repositories.

Reply:
Resolved.